database/CoreSession.java

```diff
@@ -133,6 +133,20 @@ CoreSession.Schema asSchema() {
             return this;
         }
 
+        protected CoreTransaction.Schema initialisationTransaction() {
```

Comment:
Reorder method position in the class, and change the visibility to `protected`. The logic stays unchanged.